plugin/src/queries.ts

```diff
@@ -1,395 +0,0 @@
-export const OPERATION_STATUS_QUERY = `
```

Comment:
This should probably show up as being renamed from `queries.ts` to `static-queries.ts` since some of the contents are unchanged.

Reply:
Yeah I'm not sure why git didn't detect that it was a rename.